login/service.go

```diff
@@ -9,13 +9,13 @@ import (
 )
 
 type UserData struct {
-	Name  string `form:"name"        json:"name"`
-	Email string `form:"email"       json:"email"`
-	Lang  string `form:"lang"        json:"lang"`
+	Name  string `schema:"name"        json:"name"`
+	Email string `schema:"email"       json:"email"`
+	Lang  string `schema:"lang"        json:"lang"`
 	// Either an OAuth provider or the login method
-	Provider string `form:"provider"    json:"provider"`
+	Provider string `schema:"provider"    json:"provider"`
 	// External user id, currently only used with OAuth
-	ProviderID string `form:"provider_id" json:"provider_id"`
+	ProviderID string `schema:"provider_id" json:"provider_id"`
```

Comment:
Je kan die toch weglaten voor allemaal behalve `ProviderID`? Of laat je ze expliciet staan om het uniform te houden?

Reply:
Ik denk dat dat inderdaad was om het overal uniform te houden. Ik weet niet goed in hoeverre we dit eigenlijk wel in Apollo willen definiëren of eerder voor de applicaties zouden laten. Ik heb het nu vooral aangepast omdat de oude versie blijkbaar niet meer werkte sinds we die andere schema library gebruiken